UnitySetup/UnitySetup.psm1

```diff
@@ -888,6 +993,19 @@ function Request-UnitySetupInstaller {
     }
 }
 
+function StartProcessWithArgs($startProcessArgs)
+{
+    $process = Start-Process @startProcessArgs
+    if ( $process ) {
+        if ( $process.ExitCode -ne 0) {
```

Comment:
Should force passing `-Wait` if you're going to check the exit code.

Reply:
Maybe `Invoke-Process` instead and force the `Wait`.

Reply:
There is already `-Wait` in each `$startProcessArgs`, but yes, it would be better to have it here.

Reply:
Right, only important now that it's a function that others might come along and use.